src/pages/DomainsHome/index.tsx

```diff
@@ -6,15 +6,15 @@ import { Container } from '../../components/Container';
 import { DomainItem } from '../../components/DomainItem';
 import { ControlBar } from '../../components/domains/ControlBar';
 import { NoENSBanner } from '../../components/domains/NoENSBanner';
-import { Loader } from '../../components/Loader';
+import { Loader, LoaderWrapper } from '../../components/Loader';
```

Comment:
Spinner or Loader? Let's choose one. I see `src/pages/CreateNimiPage/CreateNimiContainer.tsx` is importing Spinner

Reply:
I used loader here because something data will be refetching for milisecond and backdrop would just make screen blip for a second it looks off-putting...